rust/src/extensions.rs

```diff
@@ -33,6 +33,14 @@ pub struct Extension {
     architectures: Option<Vec<String>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     match_base_evr: Option<String>,
+    kind: ExtensionKind,
```

Comment:
IOW I think just a `#[serde(default = "ExtensionKind::OsExtension")]` would work (or `impl Default for ExtensionKind`).

Reply:
Thanks, good tip! I went with `impl Default` since it's more generic.